src/log.cr

```diff
@@ -57,8 +57,9 @@
 # A `Log` will emit the messages to the `Log::Backends` attached to it as long as
 # the configured severity filter `level` permits it.
 #
-# Logs can also be created from a type directly. The full name without generic arguments
-# will be used as a source. So for the type `Foo::Bar(Baz)` the source `foo.bar` will be used.
+# Logs can also be created from a type directly. For the type `DB::Pool` the source `db.pool` will be used
+# The full name without generic arguments will be used as a source. So for the type `Foo::Bar(Baz)`
+# the source `foo.bar` will be used.
```

Comment:
nit: The order of these sentences are weird: example, explanation, example.

Reply:
done